features/achordion.c

```diff
@@ -112,6 +109,20 @@ bool process_achordion(uint16_t keycode, keyrecord_t* record) {
     if (is_tap_hold && record->tap.count == 0 && record->event.pressed &&
         is_key_event) {
       // A tap-hold key is pressed and considered by QMK as "held".
+#ifdef ACHORDION_STREAK
+      const uint16_t s_timeout = achordion_streak_timeout(keycode);
+      // Apparently I don't know how timer_expired wortks. Can't get it to work here.
+      //if (!timer_expired(record->event.time, streak_timer + s_timeout)) {
```

Comment:
Not sure why this doesn't work. Something to do with me not understanding timers.

Reply:
Solved. Needed more parens around the addition.